rancher2/structure_cluster_v2_rke_config_machine_pool_test.go

```diff
@@ -97,6 +105,10 @@ func init() {
 				"label_one": "one",
 				"label_two": "two",
 			},
+			"machine_labels": map[string]interface{}{
```

Comment:
Fix bracket spacing in tests (same as above) ^

Reply:
I looked at this, and I couldn't figure out where or how the spacing is wrong. It seems to be consistent with this and other files in this repo?